src/main/resources/rules/rulesDefault.json

```diff
@@ -49,18 +58,65 @@
         "translation": {
           "function": "set_value",
           "parameters": {
-            "value": "f"
+            "value": " "
           }
         }
       }
     ]
   },
   {
-    "tag": "001",
-    "description": "The human readable ID",
+    "tag": "490",
+    "description": "Series",
     "dataSources": [
       {
-        "from": "$.hrid"
+        "from": "$.series",
+        "subfield": "a"
+      },
+      {
+        "indicator": "1",
+        "translation": {
+          "function": "set_value",
+          "parameters": {
+            "value": "0"
+          }
+        }
+      },
+      {
+        "indicator": "2",
+        "translation": {
+          "function": "set_value",
+          "parameters": {
+            "value": " "
```

Comment:
it is blank by default

Reply:
actually it doesn't, while reading it checks for the charAt(0) which throws a NPE if it is empty
https://github.com/folio-org/mod-data-export/blob/a19fc78b6d8be3b61053dd085cd631eaf29ecf8b/src/main/java/org/folio/service/mapping/writer/impl/AbstractRecordWriter.java#L69

Reply:
I mean - no need to specify dataSource for ind2, it is blank already if nothing specified 
https://github.com/folio-org/mod-data-export/blob/master/src/main/java/org/folio/service/mapping/writer/fields/RecordDataField.java#L14

Reply:
That will not work either, 300 field was generated as below if datasources are removed. Anyway having it explicitly that it is blank may be more readable. We can change it going forward if needed


300 | __ | $axx, 238 p. : ill. ; 24 cm.$axxix, 224 Seiten Illustrationen